Heart Disease Classification

Sumhack opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->
# Description

(Write your answer here.)

## Related Issues or Pull Requests

#(Mention Issue Number Here.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [ ] Yes
- [ ] No

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

avinashkranjan reviewed (changes requested):
Don't Make any Changes in `README`, Follow PEP-8 Guidelines, Remove extra blank lines from the code..

Sumhack commented:
Then how shall, I describe what I have done in the project. I actually have
coded this in Jupiter notebook and downloaded as python file. So hence the
blanks are there. I have removed them all now. I have deleted the content I
had changed in README and restored it with the original one.

On Fri, Apr 23, 2021 at 12:50 PM Avinash Kr. Ranjan <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Don't Make any Changes in README, Follow PEP-8 Guidelines, Remove extra
> blank lines from the code..
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/avinashkranjan/lifeMEDeasy/pull/128#pullrequestreview-642997401>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARJB362CERWGJI7NVWZH3RTTKENSFANCNFSM43IQEZEA>
> .
>

avinashkranjan commented:
@Sumhack Create a new Readme in the directory you've pushed the files

Sumhack commented:
Done.

On Sat, Apr 24, 2021 at 12:08 PM Avinash Kr. Ranjan <
***@***.***> wrote:

> @Sumhack <https://github.com/Sumhack> Create a new Readme in the
> directory you've pushed the files
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/avinashkranjan/lifeMEDeasy/pull/128#issuecomment-826044464>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARJB367QTKIL7VLKNUXNCKDTKJRQBANCNFSM43IQEZEA>
> .
>

Sumhack commented:
I have worked on the requests you had asked me for now would you please verify and merge it

avinashkranjan commented:
> I have worked on the requests you had asked me for now would you please verify and merge it

Resolve the Conflicts

Sumhack commented:
Done. Please check now


On Fri, Apr 30, 2021 at 9:22 PM Avinash Kr. Ranjan ***@***.***>
wrote:

> I have worked on the requests you had asked me for now would you please
> verify and merge it
>
> Resolve the Conflicts
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/avinashkranjan/lifeMEDeasy/pull/128#issuecomment-830190136>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARJB367QYJRZ5OFXB7FCBZDTLLG33ANCNFSM43IQEZEA>
> .
>
